src/components/AppList.component.js

```diff
@@ -116,14 +187,29 @@ class AppList extends React.Component {
         })
     }
 
-    uploadAction(e) {
+    uploadAction = e => {
         this.fileInput.click(e)
     }
 
-    upload(e) {
+    upload = e => {
         actions.installApp(e.target.files[0], this.props.uploadProgress)
     }
 
+    appVersions = app => {
+        if (!this.props.appHub.apps) {
+            return null
+        }
+
+        const appHubData = this.props.appHub.apps.find(
+            appHubApp =>
+                appHubApp.name === app.name &&
+                (!app.developer ||
+                    appHubApp.developer.name == app.developer?.name ||
+                    appHubApp.developer.organisation == app.developer?.name)
```

Comment:
I think we need to be more strict about this - if the local app doesn't have a developer defined we shouldn't recommend any updates from the App Hub.  Particularly for core apps it is important that we verify the **organisation** from the app hub is one that only the Core team controls (we use `DHIS2` for the App Management App)